### ISSUE TITLE
_.omit for collections

### ISSUE THREAD

**Issue by ooade** [you-dont-need/You-Dont-Need-Lodash-Underscore#138]
Omitting can easily be done this way:
```js
const object = { a: 1, b: '2', c: 3 };

const omit = (obj, arr) =>
	Object.entries(obj).reduce((acc, cur) => {
		if (!arr.includes(cur[0])) {
			acc[cur[0]] = cur[1];
		}

		return acc;
	}, {});

omit(object, ['a', 'c']); // => { 'b': '2' }
```

**Comment by pozylon** [comment]
```javascript
const obj = { a: 1, b: '2', c: 3 };
const { a, c, ...objectWithoutAC } = obj;
console.log(objectWithoutAC) // => { 'b': '2' }
```

**Comment by ooade** [comment]
@pozylon what about objectWithoutA, [a - e] Object without AB, or without AD, or something else?

**Comment by ooade** [comment]
Well, spread op approach takes out the omit method. Wow!

**Comment by stevemao** [comment]
PR welcome

**Comment by ooade** [comment]
Go for it @pozylon :muscle:

**Comment by pozylon** [comment]
Your turn, sir. I didn't recognize that _.omit was not in the list and your English is better for the doc stuff :P

**Comment by chandresh** [comment]
> const obj = { a: 1, b: '2', c: 3 }; const { a, c, ...objectWithoutAC } = obj; console.log(objectWithoutAC) // => { 'b': '2' }

@pozylon @ooade 

Is there a way to use it in a function and pass an array like _.omit? Something like: 
omit(object, ['a', 'c']); // => { 'b': '2' }

**Comment by ooade** [comment]
@chandresh One way to achieve that is writing it the initial proposed omit function but IMO, I don't think you need to do that. The spread/rest operator works just fine.

**Comment by chandresh** [comment]
@ooade Yes I ended up using a version of the initially proposed one. I think having a function makes it more readable. Thanks anyway.

**Comment by mrmowji** [comment]
@pozylon I think this approach doesn't work great when we want to omit an inner property.
```js
const object = { a: 1, b: '2', c: { inner: 3, anotherInner: 5 } };
omit(object, ['a', 'c.inner']); // => { 'b': '2', c: { anotherInner: 5 } }
```

I suggest we move this method to the "warn" list instead of "error"s.
